<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKurir</class>
 <widget class="QWidget" name="FormKurir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="formGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDKurirLabel">
      <property name="text">
       <string>ID Kurir</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="iDKurirLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaKurirLabel">
      <property name="text">
       <string>Nama Kurir</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaKurirLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="telpKurirLabel">
      <property name="text">
       <string>Telp Kurir</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="telpKurirLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="wilayahLabel">
      <property name="text">
       <string>Wilayah</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="wilayahLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="alamatKurirLabel">
      <property name="text">
       <string>Alamat Kurir</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="alamatKurirLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
